packages/api/utils/businessRulesValidation.test.ts

```diff
@@ -0,0 +1,9 @@
+import { uniqueNameValidator } from "./businessRulesValidation";
+
+describe("uniqueNameValidator", function () {
+  it("should throw an error if name is not unique", async () => {
+    const name = "unique name";
+    const names = ["not unique", "unique name"];
```

Comment:
The old values are back!

Reply:
Yeah, I think something happened, because I remember accepting your changes previously.